constants.testing.toml

```diff
@@ -9,6 +9,10 @@ freeze = [1, 1]
 # We use the ID 4321 for demo purposes. Change this value before deploying!
 CHAIN_ID = 4321
 CHAIN_NAME = "TestChain"
+# We will keep only this many blocks in the EVM module.
+EVM_BLOCK_PRUNING_THRESHOLD = 1000
```

Comment:
This seems like a good first step, but if he have two separate values - we can and will end up in a situation where we keep some block where it's transactions are partially prunned.
Would be better to only define the number of blocks we keep and cascade to deleting all the transactions in prunned blocks. Could be done in the next PR.